frontend/uddhar-frontend/src/shared/components/Navbar.jsx

```
@@ -57,19 +63,34 @@ const Navbar = ({  children  }) => {
             </div>
-            <div className="flex gap-10 max-lg:hidden">
+            <div className="flex items-center gap-10 max-lg:hidden">
               {navLinks.map((link, index) => (
                 <p key={index} className="text-lg hover:text-gray-500">
                   <Link to={link.path}>{link.name}</Link>
                 </p>
               ))}
-              {isLoggedIn ? signoutLink: signIn}
+              {isLoggedIn ? 
+                <div className="relative">
+                  <div onClick={()=>{setUserMenuOpen(prev => !prev)}} className="cursor-pointer">
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)